### ISSUE TITLE
Letter P doesn't show up in some labels

### ISSUE THREAD

**Issue by voltlog** [gregdavill/KiBuzzard#78]
Using Kicad 6.0.1 Kibuzzard 1.0.0 stable, not sure if this is even a Kibuzzard bug but the letter P doesn't show up in board view, while in this arrangement "2P" or "3P". The 3D viewer does show all labels correctly
![image](https://user-images.githubusercontent.com/14857827/157395331-5e27cf1d-376c-4a8b-acd7-77c8dff470ef.png)
![image](https://user-images.githubusercontent.com/14857827/157395580-4bb0fb00-1bb8-4d5f-88a7-b705ffb81837.png)

There are also some error logs:
```
--- Logging error ---
Traceback (most recent call last):
  File "C:\Program Files\KiCad\6.0\bin\Lib\logging\__init__.py", line 1086, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "C:\Users\user\OneDrive\Documents\KiCad\6.0\3rdparty\plugins\com_github_gregdavill_KiBuzzard\plugin.py", line 100, in Run
    if dlg.ShowModal() == wx.ID_OK:
  File "C:\Users\user\OneDrive\Documents\KiCad\6.0\3rdparty\plugins\com_github_gregdavill_KiBuzzard\dialog\dialog.py", line 144, in labelEditOnText
    self.ReGeneratePreview()
  File "C:\Users\user\OneDrive\Documents\KiCad\6.0\3rdparty\plugins\com_github_gregdavill_KiBuzzard\dialog\dialog.py", line 193, in ReGeneratePreview
    self.polys = self.buzzard.generate(self.m_MultiLineText.GetValue())
  File "C:\Users\user\OneDrive\Documents\KiCad\6.0\3rdparty\plugins\com_github_gregdavill_KiBuzzard\buzzard\buzzard.py", line 51, in generate
    self.svgText = self.renderLabel(inString)
  File "C:\Users\user\OneDrive\Documents\KiCad\6.0\3rdparty\plugins\com_github_gregdavill_KiBuzzard\buzzard\buzzard.py", line 71, in renderLabel
    t.set_font(self.fontName)
  File "C:\Users\user\OneDrive\Documents\KiCad\6.0\3rdparty\plugins\com_github_gregdavill_KiBuzzard\buzzard\..\deps\svg2mod\svg2mod\svg\svg.py", line 1063, in set_font
    self.font_file = self.find_font_file()
  File "C:\Users\user\OneDrive\Documents\KiCad\6.0\3rdparty\plugins\com_github_gregdavill_KiBuzzard\buzzard\..\deps\svg2mod\svg2mod\svg\svg.py", line 1198, in find_font_file
    logging.warning("Font \"{}\" does not natively support style \"{}\" using \"{}\" instead".format(
Message: 'Font "UbuntuMono-B" does not natively support style "Regular" using "Path" instead'
Arguments: ()
```

**Comment by gregdavill** [comment]
This is Interesting. If you open up the good and bad labels in the footprint editor, do they look different? 

svg2mod is a tool embedded within KiBuzzard which will attempt to insert a cut through overlapping polygons like the letter P here.

**Comment by voltlog** [comment]
This is how a "bad" label looks like in the footprint editor:
![image](https://user-images.githubusercontent.com/14857827/157599924-edf8eff2-370a-4b65-86fd-8625a41ee028.png)

**Comment by voltlog** [comment]
I've also now tried 1.1.0 and it looks like the problem is still present. Is this something that you are able to replicate on your end?

**Comment by gregdavill** [comment]
I'm using KiCad 6.0.2, on Ubuntu. 

With 1.1.0 from the PCM I've not been able to replicate your issue. I can see my labels are generating a very similar looking polygon, here it's just not filled.

![Screenshot from 2022-03-10 17-49-47](https://user-images.githubusercontent.com/344310/157609730-817d9785-5094-491d-ac08-ce4fc82a1f4c.png)
![Screenshot from 2022-03-10 17-46-15](https://user-images.githubusercontent.com/344310/157609998-7025cfa0-29ab-4864-8939-179af0d929cf.png)

I was mistaken about the svg2mod updates, they're not included in the 1.1.0 release. The updates to svg2mod place the "cuts" vertical, which might help, if KiCad is for some reason detecting an overlapping polygon, thus not rendering the inner cutouts.

![Screenshot from 2022-03-10 17-46-02](https://user-images.githubusercontent.com/344310/157611296-0d63c6ca-54b5-4100-ae77-aae5c9999ffc.png)

To rule this out can you try out the current git version? I've created a PCM package here: 
 - Uninstall your current KiBuzzard in the PCM
 - Use "install from file", to install this version: [KiBuzzard-1.2.0-pcm.zip](https://github.com/gregdavill/KiBuzzard/files/8221257/KiBuzzard-1.2.0-pcm.zip)

**Comment by voltlog** [comment]
Using Kicad 6.0.1 on Windows 11 here and with the PCM package you created the problem is no longer present.
![image](https://user-images.githubusercontent.com/14857827/157857225-fe0ee31e-eccd-4e6d-b0ef-e095d5979d72.png)
